src/BizHawk.Client.Common/DisplayManager/SimpleGDIPDisplayManager.cs

```diff
@@ -20,7 +20,7 @@ private SimpleGDIPDisplayManager(Config config, IEmulator emuCore, IGL_GDIPlus g
 				new GDIPlusGuiRenderer(glImpl))
 			{}
 
-		public SimpleGDIPDisplayManager(Config config, IEmulator emuCore, Func<(int Width, int Height)> getVirtualSize)
+		public SimpleGDIPDisplayManager(Config config, IEmulator emuCore)
```

Comment:
Don't know why this wasn't removed earlier, but this signature change will need to be propagated to testroms:
https://github.com/TASEmulators/BizHawk/blob/7acd38983ad3e64935e31daa02943fe1558e3921/src/BizHawk.Tests.Testroms/DummyFrontend.cs#L156